src/backend/database_migrations/versions/20221007_170744_nextclade_output_tables.py

```diff
@@ -0,0 +1,112 @@
+"""
+
+Create Date: 2022-10-07 17:07:52.988641
+
+"""
+import enumtables  # noqa: F401
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20221007_170744"
+down_revision = "20221004_221741"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "aligned_peptides",
+        sa.Column("sample_id", sa.Integer(), nullable=True),
+        sa.Column("entity_id", sa.Integer(), nullable=False),
```

Comment:
Just curious about using the `entity_id` field name here. We're using a different field naming convention where `uploaded_pathogen_genomes` inherits from `pathogen_genomes`

Reply:
or do we need an entity_id at all here actually? it's not in the diagram

Reply:
i had it inheriting from entity, i'll remove it since we're moving away from that